src/app/app.component.ts

```diff
@@ -49,6 +51,16 @@ export class AppComponent implements OnInit {
                 event: 'pageView',
                 url: e.urlAfterRedirects
             });
+
+            const genesRouter = e.urlAfterRedirects.match(/\(genes-router:([^)]+)\)/);
+            const pageClass = 'page-' + (
+                genesRouter && genesRouter.length > 1 && genesRouter[1]
+                ? genesRouter[1].split('/')[0]
```

Comment:
Does this need to be this complicated, to ultimately result in the css class page-gene-comparison-tool?